Sources/OpenFeature/Client/OpenFeatureClient.swift

```diff
@@ -34,13 +34,95 @@ public actor OpenFeatureClient: Sendable {
         ).value
     }
 
+    public func value(
+        for flag: String,
+        defaultingTo defaultValue: String,
+        context: OpenFeatureEvaluationContext? = nil,
+        hooks: [any OpenFeatureHook] = [],
+        hookHints: [String: OpenFeatureFieldValue] = [:]
+    ) async -> String {
+        await evaluation(
+            of: flag,
+            defaultingTo: defaultValue,
+            context: context,
+            hooks: hooks,
+            hookHints: hookHints
+        ).value
+    }
+
     public func evaluation(
         of flag: String,
         defaultingTo defaultValue: Bool,
         context: OpenFeatureEvaluationContext? = nil,
         hooks: [any OpenFeatureHook] = [],
         hookHints: [String: OpenFeatureFieldValue] = [:]
     ) async -> OpenFeatureEvaluation<Bool> {
+        await evaluation(
+            of: flag,
+            defaultingTo: defaultValue,
+            context: context,
+            hooks: hooks,
+            hookHints: hookHints,
+            performResolution: { provider, flag, defaultValue, context in
+                await provider.resolution(of: flag, defaultValue: defaultValue, context: context)
+            }
```

Comment:
Not sure if there's a nicer way to achieve the same thing 🤷‍♂️

Reply:
LGTM 👍